src/components/NcAvatar/NcAvatar.vue

```diff
@@ -559,9 +559,11 @@ export default {
 		 * @return {string}
 		 */
 		avatarUrlGenerator(user, size) {
-			let url = '/avatar/{user}/{size}'
+			const darkTheme = window.getComputedStyle(this.$el)
+				.getPropertyValue('--background-invert-if-dark') === 'invert(100%)'
```

Comment:
Reeeaaally not fond of this! :see_no_evil:

Reply:
same but I can't find a better way to do it :(

The possibly alternative I considered was using picture/img with srcset. But that would only work for setting the dark theme with media query but not with the explicit dark theme from the theming settings

Reply:
I had plans to implement a prop in the ITheme definition for each theme to specify if it's a bright or dark theme.
Didn't find the time, maybe we should for 25

That way we could use `body[data-theme-colour="dark"]` and `body[data-theme-colour="light"]` or the capabilities

Reply:
That won't work with the system theme settings since that this is only client-side and we don't know in php which theme is used.